package.json

```diff
@@ -82,7 +82,7 @@
     "sinon": "^2.3.4",
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.2",
-    "ts-loader": "^4.3.0",
+    "ts-loader": "^5.3.0",
```

Comment:
This is in `devDependencies`, so it really just means that we *test* against this version - the user can install something lower, but they will get a warning. We could also set this to `^5.0.0`, which would mean we would test against `5.0.0` in one our test suites. When the user needs to install this, they would be give the command `yarn add ts-loader ^5.0.0`, but that should install the latest version anyways.